src/components/HistoryTables/detail.tsx

```diff
@@ -180,100 +179,6 @@ DetailCaption.propTypes = {
   error: PropTypes.object
 }
 
-export const LineWrapper = styled.div`
-  display: flex;
-  flex-direction: column;
-  gap: 1em;
-
-  p,
-  hr {
-    margin: 0;
-  }
-`
-
-/**
- * Line
- * Content row of the detail page
- */
-export const Line = ({ label, depth, children }: LineProps) => (
-  <Box
-    display='flex'
-    alignItems='center'
-    gridGap='1em'
-    lineHeight='2em'
-    textAlign='left'
-    pl={`${depth * 1.5}em`}
-    css={`
-      a {
-        color: ${props => props.theme.colors.core.primary};
-        text-decoration: none;
-        &:hover {
-          text-decoration: underline;
-        }
-      }
-      .gray {
-        color: ${props => props.theme.colors.gray.medium};
-      }
-    `}
-  >
-    <Box minWidth='200px' flex='0 1 25%'>
-      {label}
-    </Box>
-    <Box
-      flex='1 1'
-      display='flex'
-      alignItems='center'
-      gridGap='0.75em'
-      style={{ wordBreak: 'break-word' }}
-    >
-      {children}
-    </Box>
-  </Box>
-)
-
-type LineProps = {
-  label?: string
-  depth?: number
-  children?: React.ReactNode
-}
-
-const LinePropTypes = {
-  label: PropTypes.string,
-  depth: PropTypes.number,
-  children: PropTypes.node
-}
-
-Line.propTypes = LinePropTypes
-Line.defaultProps = {
-  label: '',
-  depth: 0,
-  children: <></>
-}
-
-/**
- * AddressLine
- */
-
-export const AddressLine = ({ value, label, depth }: AddressLineProps) =>
-  typeof value === 'string' ? (
-    <Line label={label} depth={depth}>
-      <AddressLink address={value} hideCopyText={false} />
-    </Line>
-  ) : (
-    <Line label={label} depth={depth}>
-      <AddressLink id={value.id} address={value.robust} hideCopyText={false} />
-    </Line>
-  )
-
-type AddressLineProps = {
-  value: string | Address
-} & LineProps
-
-AddressLine.propTypes = {
-  value: PropTypes.oneOfType([PropTypes.string, ADDRESS_PROPTYPE]).isRequired,
-  ...LinePropTypes
-}
-
```

Comment:
@Schwartz10 The `LineWrapper` and `Line` seem to be useful in other contexts and are Layout related, so I moved them to `Layout`